lxd/endpoints/network.go

```diff
@@ -4,9 +4,11 @@ import (
 	"crypto/tls"
 	"fmt"
 	"net"
+	"strings"
 	"sync"
 	"time"
 
+	"github.com/armon/go-proxyproto"
```

Comment:
line break after this one as it's an external import so should be separate from our internal ones.